server/src/main/java/org/elasticsearch/common/geo/GeoJson.java

```diff
@@ -228,8 +228,12 @@ private XContentBuilder coordinatesToXContent(Polygon polygon) throws IOExceptio
 
     private static Geometry createGeometry(String type, List<Geometry> geometries, CoordinateNode coordinates, Boolean orientation,
                                            boolean defaultOrientation, boolean coerce, DistanceUnit.Distance radius) {
-
-        ShapeType shapeType = ShapeType.forName(type);
+        ShapeType shapeType;
+        if ("bbox".equals(type.toLowerCase(Locale.ROOT))) {
```

Comment:
would you mind explaining this? I do not see too much test coverage for the "bbox" situation.

Reply:
We support it in ShapeBuilder-based [GeoJson parser](https://github.com/elastic/elasticsearch/blob/df4a8147e04b4c1424414170873b2ec008905a05/server/src/main/java/org/elasticsearch/common/geo/GeoJson.java#L232) and test it [here](https://github.com/elastic/elasticsearch/blob/5fb55f62be68add6530cc883e317cb764bf4ad1e/server/src/test/java/org/elasticsearch/search/geo/GeoShapeIntegrationIT.java#L146), but I will add an additional unit-level test for it to GeoJson parser.